xml/docu_styleguide.structure.xml

```diff
@@ -1022,6 +1022,15 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
       apply a proportionally larger maximum window size.
      </para>
     </listitem>
+    <listitem>
+     <para>
+      Size screenshots correctly with the <literal>witdh</literal> attribute to
```

Comment:
Use better `<tag class="attribute">width</tag>` instead of `<literal>`.

Reply:
(Note also that Toms has already corrected the typo in the word "width"...